recipes/py_gd/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.1.6" %}
+{% set libgd_version = "2.2.2" %}
+
+package:
+    name: py_gd
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/NOAA-ORR-ERD/py_gd.git
+    git_tag: v{{ version }}
+
+build:
+    skip: True  # [py3k]
```

Comment:
> +    skip: True  # [py3k]

Yep, please change to the following.

  skip: true  # [py3k or win]


Py3k is known not to work. Windows works fine, as long as there is a libgd
build available.

The current conda-forge recipe does not support Windows, but I'm hoping to
change that.

So it doesn't seem
Like I should hard code the skip in this recipe.

-CHB



You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/conda-forge/staged-recipes/pull/1086/files/e15a7c4670f156270cc51d224adf212e421c72d2#r71091443>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AA38YEdo8CjSJbEOgO2zdnSw9YSiCEeFks5qWrHcgaJpZM4JOTtB>
.